### PR TITLE
[MTG-964] Ownership model fix using a history of owners

### PR DESCRIPTION
**Pull request by StanChe**
adding a history of owners for an asset, tracking any ownership change to determine the correct owner

As a next steps we could consider storing any owner change, so cNFTs will also have the history available.

### CODE REVIEW

File: grpc/src/mapper.rs [hunk 1]
```diff
@@ -191,6 +191,8 @@ impl TryFrom<AssetDetails> for CompleteAssetDetails {
                 .transpose()?
                 .ok_or(GrpcError::MissingField("owner_type".to_string()))?,
             owner_delegate_seq,
+            is_current_owner: todo!(),
```
Comment (kstepanovdev) [inline grpc/src/mapper.rs:194]: I suppose it will panic on `try_into()` conversion

Comment (StanChe) [reply grpc/src/mapper.rs:194]: Yep, grpc will panic: https://linear.app/mplx/issue/MTG-977/propagate-ownership-changes-to-grpc made a ticket for that

File: rocks-db/src/flatbuf/asset.fbs [hunk 2]
```diff
@@ -175,6 +172,7 @@ table AssetCompleteDetails {
   authority: AssetAuthority;
   owner: AssetOwner;
   collection: AssetCollection;
+  other_known_owners: [AssetOwner];
 }
 
 root_type AssetCompleteDetails;
```
Comment (kstepanovdev) [inline rocks-db/src/flatbuf/asset.fbs:178]: ultra nit: newline

**Review by kstepanovdev — APPROVED** [review]
Found no problems in syntax, lgtm

**Pull request merged**